src/test/java/com/google/devtools/build/lib/skyframe/rewinding/RewindingTest.java

```diff
@@ -49,12 +57,15 @@ public final class RewindingTest extends BuildIntegrationTestCase {
 
   private final ActionEventRecorder actionEventRecorder = new ActionEventRecorder();
   private final RewindingTestsHelper helper = new RewindingTestsHelper(this, actionEventRecorder);
+  private WorkerInstance worker;
 
   @Override
   protected BlazeRuntime.Builder getRuntimeBuilder() throws Exception {
     return super.getRuntimeBuilder()
+        .addBlazeModule(new RemoteModule())
```

Comment:
I can move all of this over to the helper if that makes it easier to merge. How is the Blaze version of this test structured?